pkgs/development/python-modules/astroquery/default.nix

```diff
@@ -6,25 +6,27 @@
 , keyring
 , beautifulsoup4
 , html5lib
+, numpy
 , pytest
 , pytest-astropy
+, pytest-astropy-header
 , astropy-helpers
 }:
 
 buildPythonPackage rec {
   pname = "astroquery";
-  version = "0.3.10";
+  version = "0.4";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1ce57a8792c7d5d74206d797d379de6da35d56be433ea5840c41a49f202e2fab";
+    sha256 = "1ddwnj9vpvxkrfb45c4pwv5f5za9kn2q0040dpw2ymj2bwlpl61h";
   };
 
-  propagatedBuildInputs = [ astropy requests keyring beautifulsoup4 html5lib ];
+  propagatedBuildInputs = [ astropy requests keyring beautifulsoup4 html5lib numpy ];
 
   nativeBuildInputs = [ astropy-helpers ];
 
-  checkInputs = [ pytest pytest-astropy ];
+  checkInputs = [ pytest pytest-astropy pytest-astropy-header ];
```

Comment:
`pytest-astropy` is a meta-package that installs all the dependencies used by astropy affiliated packages for testing. The latest version of `pytest-astropy` depends on `pytest-astropy-header`, but the version we have in `nixpkgs` is outdated. Instead of listing `pytest-astropy-header` here, it would be better to upgrade `pytest-astropy` in `nixpkgs`.

Reply:
Done. Thanks.